site/en/blog/chromium-chronicle-20/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "The Chromium Chronicle #20: Benchmarking Test Harnesses"
+description: >
+  Adding benchmarks is an easy way to prevent performance regressions and
+  help improve performance over time.
+layout: 'layouts/blog-post.njk'
+date: 2021-04-30
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 20:** by John Chen in Bellevue, WA (April, 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+Speed is one of Chrome's four core principles. **Adding benchmarks is an
+easy way to prevent performance regressions and help improve performance
+over time.** Good Benchmarks have a fast iteration cycle, can catch
+performance regressions much earlier than UMA, and are great for measuring
+new feature's performance.
+
+**Benchmarks run regularly in the lab.** When a regression is found, bisect
+automatically finds the culprit CL and a bug is assigned to the CL owner.
+
+**Chrome benchmarks combine sequences of web page interactions (called
+stories) with performance measurements.** Similar cases are grouped into
+benchmark harnesses. New benchmarks usually fit into one of the
+[existing harnesses](http://bit.ly/chrome-benchmark-harnesses):
+
+{% Columns %}
+
+{% Column %}
+
+* System Health
+* Loading
+* Memory
+* Rendering
+* Power
+* Startup
+
+{% endColumn %}
+{% Column %}
+
+* V8 Runtime
+* Media
+* WebRTC
+* Press
+* Blink Perf
+
+{% endColumn %}
+{% endColumns %}
+
+**The Telemetry framework replays recorded stories to simulate user
+interactions with Chrome while collecting traces that record Chrome
+activities.** After the stories finish, the framework runs various performance
+metrics to analyze the traces and calculates performance results.
+
+You can cover most new performance test cases in Chrome by adding a new story,
+using an existing metric within one of the existing harnesses. You can also
+collect additional traces and add more metrics to existing benchmarks or pass
+additional flags to the browser.
+
+Use Blink Perf for one off cases that don't fit into other harnesses. In
```

Comment:
Nit: one off --> one-off to stay consistent with the other usage of "one-off" spelling.